lib/Github/Api/Apps.php

```diff
@@ -9,6 +9,13 @@
  */
 class Apps extends AbstractApi
 {
+    use AcceptHeaderTrait;
+
+    public function configure()
```

Comment:
Can you make the configure method private and call it in each api method that needs the custom header. This way it's easier to remove it when the beta period is over without BC breaks.

Reply:
Done!